Content.Server/Botany/SeedPrototype.cs

```diff
@@ -59,6 +59,8 @@ public partial struct SeedChemQuantity
     [DataField("Min")] public int Min;
     [DataField("Max")] public int Max;
     [DataField("PotencyDivisor")] public int PotencyDivisor;
+    [DataField("Inherent")] public bool Inherent = true;
```

Comment:
Please start documenting new fields with the explanations you wrote in "Technical Details". It should be easier for someone reading the source in the future to understand what these new fields mean/do.

Reply:
fixed